pkg/controller/cronjob/utils.go

```diff
@@ -150,12 +150,12 @@ func getRecentUnmetScheduleTimes(cj batchv1beta1.CronJob, now time.Time) ([]time
 	return starts, nil
 }
 
-// getUnmetScheduleTimes gets the slice of all the missed times from the time a job
+// getNextScheduleTime gets the slice of all the missed times from the time a job
```

Comment:
Nit: the doc says about slice of times, but that's not true any more, this returns a single time time pointer now.

Reply:
updated, PTAL